mod/research/set/self/research_page/_year_option.haml

```diff
@@ -1,11 +1,12 @@
-%div
-  %h6.d-flex.flex-no-wrap.text-muted.mb-0
-    %div.mr-5
-      %strong
-        = item
-    -if record_card.answers_by_year[item]
-      %div
-        = nest record_card.answers_by_year[item], view: :conciser, hide: :year_equals
-    -else
-      %i
-        add answer
\ No newline at end of file
+- if item
```

Comment:
In dropdown, first year wasn't selected and so wasn't triggering change.  This let me add a blank year.